includes/class-freemius.php

```diff
@@ -14823,15 +14823,42 @@ function has_affiliate_program() {
             return $this->_plugin->has_affiliate_program();
         }
 
+        /**
+         * Get Plugin ID under which we will track affiliate application.
+         *
+         * This could either be the Bundle ID or the main plugin ID.
+         *
+         * @return number Bundle ID if developer has provided one, else the main plugin ID.
+         */
+        private function get_plugin_id_for_affiliate_terms() {
+            $bundle_id = $this->get_bundle_id();
+            if ( ! is_null( $bundle_id ) ) {

```

Comment:
@swashata for short `if/else` returns I like using `?:`:
```
    return is_null( $bundle_id ) ?
        $this->_plugin->id :
        $bundle_id;
```

I think it improves readability.